app/Config.hs

```diff
@@ -16,10 +17,8 @@ import System.Hapistrano.Commands
 data Config = Config
   { configDeployPath :: !(Path Abs Dir)
     -- ^ Top-level deploy directory on target machine
-  , configHost :: !(Maybe String)
-    -- ^ Host to deploy to. If missing, localhost will be assumed.
-  , configPort :: !Word
-    -- ^ SSH port number to use, may be omitted
+  , configHosts :: ![(String, Word)]
```

Comment:
`Map String Word` is OK, could be `Set (String, Word)` but then the same host with different ports will be allowed.